### ISSUE TITLE
Error in Parse+LiveQuery.swift : Cannot call value of non-function type 'ParseClientConfiguration?'

### ISSUE THREAD

**Issue by loxx123** [parse-community/ParseLiveQuery-iOS-OSX#217]
current code 

```
extension Parse {
    static func validatedCurrentConfiguration() -> ParseClientConfiguration {
        guard let configuration = Parse.currentConfiguration() else {
            preconditionFailure("Parse SDK is not initialized. Call Parse.initializeWithConfiguration() before loading live query client.")
        }
        return configuration
    }
}
```

error: 

> Cannot call value of non-function type 'ParseClientConfiguration?'

fixing the error:

```
extension Parse {
    static func validatedCurrentConfiguration() -> ParseClientConfiguration {
      guard let configuration = Parse.currentConfiguration else {
            preconditionFailure("Parse SDK is not initialized. Call Parse.initializeWithConfiguration() before loading live query client.")
        }
        return configuration
    }
}
```

this is in objective-c project, this won't really fix the error, however, since i'd have to rebuild the framework. let me know your thoughts on why this error is happening.

**Comment by bstillitano** [comment]
I also have this issue after updating an objective-c project to the latest SDK.

**Comment by TomWFox** [comment]
I believe a similar solution has been applied in #210

**Comment by drdaz** [comment]
This should be fixed in the new 2.7.0 release.
